src/libsync/configfile.h

```diff
@@ -42,6 +42,7 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     static QString configPath();
     static QString configFile();
     static bool exists();
+    const int DefaultRemotePollInterval { 30000 }; // default remote poll time in milliseconds
```

Comment:
Would it make sense to make 
`const std::chrono::milliseconds DefaultRemotePollInterval{30000};` instead of int?
There are 2 places where DefaultRemotePollInterval is used and both of them create std::chrono::milliseconds instance.

Reply:
Thanks, done that!